winsparkle.pas

```diff
@@ -0,0 +1,413 @@
+ {
+   *  This file is part of WinSparkle (https://winsparkle.org)
+   *
+   *  Copyright (C) 2009-2016 Vaclav Slavik
+   *
+   *  Permission is hereby granted, free of charge, to any person obtaining a
+   *  copy of this software and associated documentation files (the "Software"),
+   *  to deal in the Software without restriction, including without limitation
+   *  the rights to use, copy, modify, merge, publish, distribute, sublicense,
+   *  and/or sell copies of the Software, and to permit persons to whom the
+   *  Software is furnished to do so, subject to the following conditions:
+   *
+   *  The above copyright notice and this permission notice shall be included in
+   *  all copies or substantial portions of the Software.
+   *
+   *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   *  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+   *  DEALINGS IN THE SOFTWARE.
+   *
+ }
+//Converted to Pascal by Theodore Tsirpanis.
+unit winsparkle;
+
+{$IFNDEF MSWINDOWS}
+{$FATAL WinSparkle requires Windows}
+{$ENDIF}
+
+interface
+
+uses
+  ctypes;
+
+const
+  WinSparkleLib = 'winsparkle.dll';
+
+  {--------------------------------------------------------------------------*
+                         Initialization and shutdown
+   *-------------------------------------------------------------------------- }
+  {*
+      @name Initialization functions
+    }
+
+  {*
+      Starts WinSparkle.
```

Comment:
Is there any reason to keep these comments here? I.e. does the IDE use them? If not, then I’d prefer to remove them from this file (they are already in the C header — I’ll add a doxygen-generated copy to the site to make accessing the docs easier) and keep the file small, with just the declarations.